components/Account/Memberships/Claim.js

```diff
@@ -213,9 +258,16 @@ class ClaimMembership extends Component {
 
     return (
       <div>
-        <H2 style={{ marginBottom: 20 }}>
-          {t('memberships/claim/lead')}
-        </H2>
+        {contextLead &&
+          <H2 style={{ marginBottom: 20 }}>
+            {contextLead}
+          </H2>
+        }
+        {contextBody &&
+          <RawHtml type={P} dangerouslySetInnerHTML={{
+            __html: contextBody
```

Comment:
Just because of yesterday: if contextBody comes in via the `context` query, is this potentially unsafe?

Reply:
It's safe.